resolution/src/main/java/net/digitalid/core/resolution/IdentifierResolverImplementation.java

```diff
@@ -153,10 +150,7 @@ public static void initializeIdentifierResolver() throws DatabaseException {
             } else if (identifier instanceof InternalNonHostIdentifier) {
                 final @Nonnull InternalNonHostIdentifier internalNonHostIdentifier = (InternalNonHostIdentifier) identifier;
                 final @Nonnull HostIdentifier hostIdentifier = internalNonHostIdentifier.getHostIdentifier();
-                if (Host.exists(hostIdentifier)) {
```

Comment:
Why do we not need this check anymore?

Reply:
It's only/mainly an issue for tests that run the client and the host in the same process and run into locking issues. The main reason for removing it is that we can no longer do this check because `core-resolution` no longer depends on `core-host`. The test that caused the issues has already been disabled before.